e2e/snapshot-creators/default.cy.snap.js

```diff
@@ -132,19 +164,40 @@ describe("snapshots", () => {
 
     cy.updatePermissionsGraph({
       [ALL_USERS_GROUP]: {
-        [SAMPLE_DB_ID]: { data: { schemas: "none", native: "none" } },
+        [SAMPLE_DB_ID]: {
+          // set the data permission so the UI doesn't warn us that "all users has higher permissions than X"
```

Comment:
The UI is still based on data permissions (for now!) so I'm setting all those just as before.